api/nodes/serializers.py

```diff
@@ -75,7 +75,7 @@ def update_institutions(node, new_institutions, user, post=False):
         new_institutions=new_institutions,
     )
 
-    if post and not len(add):
+    if post and not len(add) and node.affiliated_institutions.count() == 0:
         raise RelationshipPostMakesNoChanges
```

Comment:
I wonder if it would make more sense just to take this out. I don't see it being particularly useful.